README.rst

```diff
@@ -151,10 +151,36 @@ General
     logger = logging.getLogger("cli.logger")
     logger.info('hi')
 
-**Notes**: - All loggers set up and created before the initialization of the Cloud Foundry logging library will
-be left untouched. - When using Flask and Sanic with the logging library a before and
+**Notes**: All loggers set up and created before the initialization of the Cloud Foundry logging library will
+be left untouched. When using Flask and Sanic with the logging library a before and
 after request middleware is attached, and it will capture response times for each request.
 
+Setting and getting correlation ID
+""""""""""""""""""""""""""""""""""
+
+When using cf_logging in a web application you don't need to care about setting and getting correlation ID,
+because the logging frameworks supported will fetch it from HTTP headers and will set it to the log output for you.
```

Comment:
because the logging frameworks supported will fetch it from HTTP headers and will set it to the log output for you. -> because the logging library will fetch it from the HTTP headers and set it.